Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM heroku/cedar:14
+
+RUN useradd -d /app -m app
+USER app
+WORKDIR /app/src
+
+ENV HOME /app
+ENV PORT 3000
+ENV PATH /app/.heroku/python/bin:/tmp/python-pack/bin:$PATH
+ENV STACK cedar-14
+ENV PYTHONHOME /app/.heroku/python
+ENV PYTHONPATH /app/
+ENV DOCKER_BUILD 1
+
+
+RUN mkdir -p /app/.heroku
```

Comment:
I recommend merging these together into a big chain (ie `mkdir -p /app/.heroku && mkdir -p /tmp/app && mkdir -p /app/src && etc` in order to avoid creating a large number of intermediate images when building.